### PR TITLE
PR for test  Review code Procces (self review)

### PR DESCRIPTION
**Pull request by ivanmvh**

### CODE REVIEW

**Comment by ivanmvh** [comment]
Please do you have linters errors. Remember that a norm is that you cant ask for a review if you have linters errors. 🙏 
image

**Review by ivanmvh — COMMENTED** [review]
Please your code have linters errors. Remember you can't ask for a review if you had linters errors.
image.

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
   </head>
 
   <body>
-    we are a microverse programer very bad 
+    We are a Microverse programer - Thechnical Documentation Page (Project)
```
Comment (ivanmvh) [inline index.html]: A text must be enclosed between tags. For example <h1></h1> or h2, h3 o p.

**Review by ivanmvh — COMMENTED** [review]
Please do the changues required.

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: > A text must be enclosed between tags. For example
> 
> or h2, h3 o p.

- [x] Good work using inline comments 👍 . However, Kindly try to explain the reason as to `why?` you are requesting this change. The would enable the other person understand you clearly
- [x] Kindly provide at least two reviews.

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @ivanmvh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ Correct use of Github inline comments


## Required Changes ♻️
- [ REQUIRED] Kindly provide a description to your Pull Request Describing what you did. This is important for documentation purposes
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ivanmvh** [comment]
STATUS: REQUIRED CHANGES ♻️
Hi @ivanmvh 👋

Your project page is looking fantastic so far 🤩 !

However, there is still some room for improvement, that can help you have better code and make your project professional 😎

**Comment by ivanmvh** [comment]
Highlights ✨
Great work using the right GitHub Flow 💯

Commits are descriptive and meaningful 👍
Project Documentation (PR, README) is professional 👔
Congrats on your work so far! 👏

Required Changes ♻️
There are some issues that you still need to work on to go to the next project, but you are almost there!
For more information, check the comments under the review ⬇️

Optional suggestions ☑️
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 💯

Happy coding and keep it up! 🙌

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Nyame-Wolf in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
...
     <main id="main-doc">Hello
```
Comment (ivanmvh) [inline index.html:11]: I suggest you fix the margins of the main section. Currently your content seems squeezed yet you have a lot of empty space left.

File: index.html [hunk 3]
```diff
@@ -20,7 +20,7 @@
...
       <section id="topic1" class="main-section">
         <header>topic1 </header>
```
Comment (ivanmvh) [inline index.html:13]: Kindly note that it is a requirement that you match the design as much as you can. Currently the cards in this section have too much margins. Please fix this:
image

**Review by ivanmvh — COMMENTED** [review]
STATUS: REQUIRED CHANGES ♻️
Hi @ivanmvh 👋

Your project page is looking fantastic so far 🤩 !

However, there is still some room for improvement, that can help you have better code and make your project professional 😎

Highlights ✨
Great work using the right GitHub Flow 💯
Great work using the same font as Figma design ☑️
Commits are descriptive and meaningful 👍
Project Documentation (PR, README) is professional 👔
Congrats on your work so far! 👏

Required Changes ♻️
There are some issues that you still need to work on to go to the next project, but you are almost there!
For more information, check the comments under the review ⬇️

Optional suggestions ☑️
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 💯

Happy coding and keep it up! 🙌

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @ivanmvh  in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @ivanmvh , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that Linters not passed

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ivanmvh** [comment]
STATUS: REQUIRED CHANGES ♻️
Hi @ivanmvh 👋

Your project page is looking fantastic so far 🤩 !

However, there is still some room for improvement, that can help you have better code and make your project professional 😎

Highlights ✨
Great work using the right GitHub Flow 💯
Great work using the same font as Figma design ☑️
Commits are descriptive and meaningful 👍
Project Documentation (PR, README) is professional 👔
Congrats on your work so far! 👏

Required Changes ♻️
There are some issues that you still need to work on to go to the next project, but you are almost there!
For more information, check the comments under the review ⬇️

Optional suggestions ☑️
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code 💯

Happy coding and keep it up! 🙌

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @ivanmvh in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  --black: #000;
+  --wine: #730d15;
+  --red: #f00;
+  --gray: gray;
+  --gray2: #262626;
+  --white: #fff;
+}
+
+#navbar {
+  /* position: absolute */
```
Comment (ivanmvh) [inline css/styles.css:11]: You should not leave unused lines of code as comments unless there is a reason why that commented code is there. If there is no valid reason delete those comments that make your code not clean. 🧹

File: index.html [hunk 5]
```diff
@@ -1,26 +1,13 @@
...
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="stylesheet" media="(max-width: 800px)" href="example.css" />
-  </head>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
 
-  <body>
-    we are a microverse programer very bad 
+<body>
+    We are a Microverse programer - Thechnical Documentation Page (Project) -ivmvh
```
Comment (ivanmvh) [inline index.html:10]: This text must be enclosed in some class of tag depending of his semantic meaning.  If is a Title could be <p> y is a title use h1,h2 or h3.
https://www.w3schools.com/html/html5_semantic_elements.asp

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @ivanmvh     ,

### Status: Approved ✔️ ✔️ ✔️

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏
## Optional Suggestion

- [ ] Please make sure that your PR has description about what changes and activity you made in detail.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**